dashboard/app/views/home/_gallery.html.haml

```diff
@@ -1,9 +1,30 @@
+-show_heading = local_assigns[:show_heading]
+
 .course_progress
-  .row#lesson
-    .span12#heading
-      %h3.whitetext{style: 'margin: 0'}
-        = t("home.your_gallery")
+  -if show_heading
+    .row#lesson
+      .span12#heading
+        %h3.whitetext{style: 'margin: 0'}
+          = t("home.your_gallery")
   .row#stats
     .span12{style: 'margin-left: 10px;'}
       %div.gallery#gallery
         = render partial: 'home/gallery_content'
+
+:javascript
+  $(document).ready(function() {
```

Comment:
For background, my reasons for not doing this initially were that this code is (1) being moved from another haml file, and (2) going away just as soon as we've migrated to the new project gallery within the next 2-3 sprints. I'm fine with extracting if we don't want to set a bad example though.